src/fixup.js

```diff
@@ -297,6 +297,40 @@
     request.send();
   }
 
+  /* Dark mode toggle */
+  const darkCss = document.querySelector('link[href="dark.css"]');
```

Comment:
This needs to be $= otherwise it'll never match except for the README!

Reply:
oops, yes.

Reply:
addressed: https://github.com/w3c/tr-design/pull/243/commits/9efe263c396ab35f998f9d313532176e6851367e

Reply:
Note that I would expect the link to be something like https://www.w3.org/StyleSheets/TR/2016/dark.css once the PR is merged